src/charts/stacked-bar.js

```diff
@@ -411,46 +413,42 @@ define(function(require){
 
         /**
          * Draws the bars along the x axis
-         * @param  {D3Selection} bars Selection of bars
+         * @param  {D3Selection} layersSelection Selection of bars
          * @return {void}
          */
-        function drawHorizontalBars(series) {
+        function drawHorizontalBars(layersSelection) {
+            let layerJoin = layersSelection
```

Comment:
new repo! any interest in switching to const when possible?

Reply:
It's not on my list of priorities, although I know Jaylum is more used to use const.